docs/topics/link.rst

```diff
@@ -0,0 +1,44 @@
+.. _topics-link:
+
+====
+Link
+====
+
+Link objects represent an extracted link by the :class:`~scrapy.linkextractors.lxmlhtml.LxmlLinkExtractor`.
+
+The ``__init__`` method of
+:class:`scrapy.link.Link` takes values that describe structure of the anchor tag that makes
+up the link. :class:`LxmlLinkExtractor.extract_links
+<scrapy.linkextractors.lxmlhtml.LxmlLinkExtractor.extract_links>` returns a
+list of matching :class:`scrapy.link.Link` objects from a
+:class:`~scrapy.http.Response` object.
+
+
+Link
+----
+
+.. module:: scrapy.link
+   :synopsis: Link from link extractors
+
+
+.. class:: Link(url, text='', fragment='', nofollow=False)

```

Comment:
Could `autoclass` be used here?

Reply:
I'm pretty new to documenting with Sphinx. I'll take a look at what an **autoclass** is and get back to you.

Reply:
The docstring for the Link class only contains the words """Link objects represent an extracted link by the LinkExtractor.""" 

I don't think that suffices, do you?

Reply:
It does not. But you _can_ move your documentation to the docstring, and _then_ use `autoclass` :slightly_smiling_face:

Reply:
Done. @Gallaecio
Thanks for the suggestion @elacuesta I see it's a better option.